modules/configure-an-additional-clientCA.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * security/certificates/api-server.adoc
+
+:_content-type: PROCEDURE
+[id="configure-an-additional-clientCA-for-the-OpenShift-API-server_{context}"]
+
+= Configure an additional clientCA for the OpenShift API server
+
+Optionally, you may choose to invalidate the installer-generated kubeconfig. You would do this when:
+
+* You don't trust who installed the cluster
+* The kubeconfig is leaked
```

Comment:
```suggestion
* The kubeconfig is leaked.
```

Reply:
Not sure what this means, maybe just my ignorance.

Reply:
- Added periods at the end of the listed items.
- I don't know how a kubeconfig could be leaked, either.